packages/lib/src/components/Card/components/CardInput/CardInput.scss

```
@@ -108,3 +108,23 @@
         transition: none;
     }
 }
+
+.adyen-checkout__fieldset--revolving-plan{
+    .adyen-checkout__radio_group{
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.adyen-checkout__fieldset--revolving-plan{
+    .adyen-checkout__radio_group__input-wrapper{
+        margin-top: 20px
+    }
+}
```

```suggestion
.adyen-checkout__fieldset--revolving-plan {
    .adyen-checkout__radio_group {
        display: flex;
        flex-direction: column;
    }

    .adyen-checkout__radio_group__input-wrapper {
        margin-top: 20px
    }
}
```

There's a duplicate CSS selector here